var/router-config.json

```diff
@@ -5,6 +5,9 @@
       "removeItinerariesWithSameRoutesAndStops": "${REMOVE_ITINERARIES_WITH_SAME_ROUTES_AND_STOPS}"
     },
     "searchWindow": "${SEARCH_WINDOW}",
+    // "unpreferred": {
+    //   "agencies": ["mbta-ma-us-no-disruptions:1"]
+    // },
```

Comment:
Let's delete this

Reply:
[Byeee](https://github.com/mbta/otp-deploy/pull/42/commits/9d3d3c0444ea98070dad2cfda02a254d4be89b57)